tests/README.md

```diff
@@ -102,48 +102,16 @@ The pytests are tests that will execute small playbooks and then will verify the
 To select only these tests on a test execution use the option `-m "not playbook"`.
 
 
-## Running tests in a docker container
+## Running tests in a container
 
-It's also possible to run the tests in a container.
-
-### Creating a container to run the tests
-
-Before setting up a container you will need to install molecule framework:
-
-```
-pip install molecule-plugins[docker]
-```
-
-Now you can start a test container using the following command:
-```
-molecule create -s c8s
-```
-
-Note: Currently the containers available for running the tests are:
- * fedora-latest
- * centos-7
- * c8s
- * c9s
-
-### Running the tests inside the container
+It's also possible to run the tests in a container. Use the script `infra/image/start.sh` to start a container.
 
 To run the tests you will use pytest (works the same as for VMs).
 
 ```
-RUN_TESTS_IN_DOCKER=1 IPA_SERVER_HOST=c8s pytest
+RUN_TESTS_IN_DOCKER=podman IPA_SERVER_HOST=ansbile-freeipa-tests pytest -m "playbook"
```

Comment:
It might be good to show also the use of the container without pytest:

`ansible-playbook -i infra/image/inventory tests/location/test_location.yml`

As an example.

Reply:
This file should be completely reviewed, in my opinion.

We should start with "higher level" methods for testing (start.sh + playbooks and run_tests.sh) and then show what can be done with "lower level" commands (e.g. pytest).